planner/optimizer/optimizer-core/src/test/java/eu/seaclouds/platform/planner/optimizerTest/OptimizerTOSCASeptember2015Test.java

```diff
@@ -32,201 +32,228 @@
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
+import org.yaml.snakeyaml.Yaml;
 
 import eu.seaclouds.platform.planner.optimizer.Optimizer;
 import eu.seaclouds.platform.planner.optimizer.heuristics.SearchMethodName;
 import eu.seaclouds.platform.planner.optimizer.util.TOSCAkeywords;
+import eu.seaclouds.platform.planner.optimizer.util.YAMLgroupsOptimizerParser;
+import eu.seaclouds.platform.planner.optimizer.util.YAMLoptimizerParser;
 
 public class OptimizerTOSCASeptember2015Test {
 
-   private static Optimizer    optimizer;
-   private static String       appModel;
-   private static String       suitableCloudOffer;
-
-   
-   static Logger               log;
+   private static Optimizer optimizer;
+   private static String appModel;
+   private static String suitableCloudOffer;
 
+   static Logger log;
 
+   @BeforeClass
+   public void createObjects() {
 
-@BeforeClass
-public void createObjects() {
+      log = LoggerFactory.getLogger(OptimizerTOSCASeptember2015Test.class);
 
+      log.info("Starting TEST optimizer for the TOSCA syntax of September 2015");
 
-   log                   = LoggerFactory
-         .getLogger(OptimizerTOSCASeptember2015Test.class);
+      final String dir = System.getProperty("user.dir");
+      log.debug("Trying to open files: current executino dir = " + dir);
 
-   log.info("Starting TEST optimizer for the TOSCA syntax of September 2015");
+      try {
+         appModel = filenameToString(TestConstants.APP_MODEL_FILENAME);
+      } catch (IOException e) {
+         log.error("File for APPmodel not found");
+         e.printStackTrace();
+      }
 
-   final String dir = System.getProperty("user.dir");
-   log.debug("Trying to open files: current executino dir = " + dir);
+      try {
+         suitableCloudOffer = filenameToString(TestConstants.CLOUD_OFFER_FILENAME_IN_JSON);
+      } catch (IOException e) {
+         log.error("File for Cloud Offers not found");
+         e.printStackTrace();
+      }
 
-   try {
-      appModel = filenameToString(TestConstants.APP_MODEL_FILENAME);
-   } catch (IOException e) {
-      log.error("File for APPmodel not found");
-      e.printStackTrace();
    }
 
-   try {
-      suitableCloudOffer = filenameToString(TestConstants.CLOUD_OFFER_FILENAME_IN_JSON);
-   } catch (IOException e) {
-      log.error("File for Cloud Offers not found");
-      e.printStackTrace();
+   private static String filenameToString(String path) throws IOException {
+      byte[] encoded = Files.readAllBytes(Paths.get(path));
+      return new String(encoded, StandardCharsets.UTF_8);
    }
 
-}
+   @Test(enabled = true)
```

Comment:
`enable = true` is default value, so you could remove it. Really, I think you could move `@Test` tag to class leve.